src/Components/Web/src/PublicAPI.Shipped.txt

```diff
@@ -140,6 +140,8 @@ Microsoft.AspNetCore.Components.Forms.ValidationSummary.AdditionalAttributes.get
 Microsoft.AspNetCore.Components.Forms.ValidationSummary.AdditionalAttributes.set -> void
 Microsoft.AspNetCore.Components.Forms.ValidationSummary.Model.get -> object?
 Microsoft.AspNetCore.Components.Forms.ValidationSummary.Model.set -> void
+Microsoft.AspNetCore.Components.Forms.ValidationSummary.CssClass.get -> string?
+Microsoft.AspNetCore.Components.Forms.ValidationSummary.CssClass.set -> void
```

Comment:
Please move these to the PublicAPI.Unshipped.txt file in the same directory.

Reply:
Sounds good, I will work on this after work today! :) Thanks!

Reply:
> Should I update my existing issue with the proposal template or should I make a new issue and link both?

Make a new issue and link anything relevant please 😃

Reply:
Hi @Nick-Stanton. Sorry it took me a bit to get to this. I have been super busy at work. :)

I moved the new api's to the `PublicAPI.Unshipped.txt` file and created an api proposal for this change here: https://github.com/dotnet/aspnetcore/issues/46461